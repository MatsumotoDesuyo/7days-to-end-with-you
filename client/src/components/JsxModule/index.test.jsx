import React from 'react';
import renderer from 'react-test-renderer';
import { JsxModule } from '.';

test('test test', () => {
  const component = renderer.create(<JsxModule />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
