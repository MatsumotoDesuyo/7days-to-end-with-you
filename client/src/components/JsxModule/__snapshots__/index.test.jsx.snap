// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test test 1`] = `
<h2>
  I&#39m JS module.
</h2>
`;
